Evaluation_1/TPU_Samples/README.md

```diff
@@ -0,0 +1,2 @@
+# Image Retraining with Tensorflow 2.0, TF Hub and Cloud TPU
```

Comment:
Also add instructions in these READMEs about stuff like high-level requirements (TPU access on Cloud, etc), commands to run, etc. This will make these scripts way more useful.
This can be done in a followup PR, just add a line here like:
```
TODO(captain-pool@): <...>
```